extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -234,3 +236,71 @@
   margin-bottom: 20px !important;
   color: var(--i-amphtml-label-text-color) !important;
 }
+
+/** For amp-story-outlink-page-attachment-v2 experiment elements. */
+.i-amphtml-story-outlink-page-attachment-arrow {
+  display: block !important;
+  height: 32px !important;
+  width: 32px !important;
+  cursor: pointer !important;
+  position: relative;
+}
+
+.i-amphtml-story-outlink-page-open-attachment-bar-left {
+  background: white !important;
+  left: 6px !important;
+  animation: open-attachment-bar-left 0.3s cubic-bezier(0.4, 0.0, 0.2, 1) both !important;
+}
+
+.i-amphtml-story-outlink-page-open-attachment-bar-right {
+  background: white !important;
+  right: 6px !important;
+  animation: open-attachment-bar-right 0.3s cubic-bezier(0.4, 0.0, 0.2, 1) both !important;
+}
+
+.i-amphtml-story-outlink-page-attachment-outlink-chip {
+  display: flex !important;
+  position: relative !important;
+  padding: 10px 14px 10px 6px !important;
+  margin: 0 0 20px 0 !important;
+  height: 16px !important;
+  max-width: 243px !important;
```

Comment:
Why this magic number?

Reply:
Hong calculated it was the width of the smallest supported screen minus the navigation margins

Reply:
She changed it to 240 to make it cleaner.

Reply:
Should we do what's currently done for the attachment open UI? We have `max-width: calc(100% - 64px) !important;`

I think it'd make sense for the UI to grow and show as much text as possible given the size of the screen?

Reply:
That works! Just manually tested it too, it crops at the same point on the small screen, but it's more clear where the number is from. Thanks!